react-router/lib/createMemoryHistory.d.ts

```diff
@@ -1,3 +1,8 @@
+// Type definitions for react-router v2.0.0
```

Comment:
I think this pattern is problematic. What happens if these headers don't agree within a folder? How do people know which to update? How is the types publisher supposed to know which version to use when generating the readme?

The header should only be required for the entry point file.